src/functionalTest/java/uk/gov/hmcts/sptribs/caseworker/CaseworkerCancelHearingFT.java

```diff
@@ -1,77 +1,119 @@
 import uk.gov.hmcts.sptribs.caseworker.util.EventConstants;
 import uk.gov.hmcts.sptribs.testutil.FunctionalTestSuite;
 
+import java.util.HashMap;
 import java.util.Map;
 
 import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
 import static net.javacrumbs.jsonunit.assertj.JsonAssertions.json;
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER;
 import static net.javacrumbs.jsonunit.core.Option.IGNORING_EXTRA_FIELDS;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.springframework.http.HttpStatus.OK;
 import static uk.gov.hmcts.sptribs.testutil.CaseDataUtil.caseData;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.ABOUT_TO_START_URL;
 import static uk.gov.hmcts.sptribs.testutil.TestConstants.ABOUT_TO_SUBMIT_URL;
 import static uk.gov.hmcts.sptribs.testutil.TestConstants.RECORD_NOTIFY_PARTIES_MID_EVENT_URL;
 import static uk.gov.hmcts.sptribs.testutil.TestConstants.SUBMITTED_URL;
 import static uk.gov.hmcts.sptribs.testutil.TestResourceUtil.expectedResponse;
 
 @SpringBootTest
-@Ignore
 public class CaseworkerCancelHearingFT extends FunctionalTestSuite {
 
-    private static final String REQUEST_SUBMIT = "classpath:request/casedata/ccd-callback-casedata-cancel-hearing-about-to-submit.json";
-    private static final String REQUEST_SUBMITTED = "classpath:request/casedata/ccd-callback-casedata-cancel-hearing-submitted.json";
-    private static final String REQUEST_MID_EVENT = "classpath:request/casedata/ccd-callback-casedata.json";
+    private static final String REQUEST_ABOUT_TO_START = "classpath:request/casedata/ccd-callback-casedata.json";
+    private static final String REQUEST_ABOUT_TO_SUBMIT =
+        "classpath:request/casedata/ccd-callback-casedata-cancel-hearing-about-to-submit.json";
+    private static final String REQUEST_SUBMITTED_ONE_PARTY =
+        "classpath:request/casedata/ccd-callback-casedata-cancel-hearing-submitted.json";
+    private static final String REQUEST_ALL_PARTIES_NULL = "classpath:request/casedata/ccd-callback-casedata.json";
+    private static final String REQUEST_ONE_PARTY_INVALID =
+        "classpath:request/casedata/ccd-callback-casedata-cancel-hearing-submitted_invalid_party.json";
+
+    private static final String RESPONSE_ABOUT_TO_START =
+        "classpath:responses/response-caseworker-cancel-hearing-about-to-start.json";
+    private static final String RESPONSE_ABOUT_TO_SUBMIT = "classpath:responses/response-caseworker-cancel-hearing-about-to-submit.json";
+
+    private static final String CONFIRMATION_HEADER = "$.confirmation_header";
+    private static final String ERROR_MESSAGE = "$.errors";
+
+    @Test
+    public void shouldInitialiseTheHearingListWhenAboutToStartCallbackIsInvoked() throws Exception {
+        final Map<String, Object> caseData = caseData(REQUEST_ABOUT_TO_START);
+
+        final Response response = triggerCallback(caseData, EventConstants.CASEWORKER_CANCEL_HEARING, ABOUT_TO_START_URL);
```

Comment:
You can import the event name as static import removing the need for EventConstants.